### ISSUE TITLE
Main title in banner should not be h1. It should just be big text

### ISSUE THREAD

**Issue by yannickwurm** [wurmlab/wurmlab.github.io#150]
Because this leads many pages to have two h1s. 

- [ ] Main title in banner should not be h1. It should just be big text
- [ ] Check (with grep?) or with checkbot.io that every page has an h1

**Comment by yuriigr** [comment]
I understand this. It was a workaround for 'SEO audit' issue, related to missing h1 on multiple pages.
I can rollback <h1> to <p> for the main banner, if needed. And use <h1> banner only for Home and News pages, because they don't have any header text at all.

**Comment by yannickwurm** [comment]
I undrestand.

I guess h1 can be added specifically inside those 2 pages (rather than in the banner?).

**Comment by yuriigr** [comment]
Yes. But these two pages was only an example of page w/o title. A lot of pages have h2 or h3 as a main title.

**Comment by yannickwurm** [comment]
OK yeah - so I guess we want to upgrade those titles to h1

**Comment by yuriigr** [comment]
Ok, no problem. I'm going to undo the banner changes. After that we will run the SEO and Checkbot.
Pls, provide me with titles for Home and News pages.
